app/services/forest_liana/permissions_checker.rb

```diff
@@ -27,11 +28,33 @@ def get_last_retrieve
         @@permissions_per_rendering[@rendering_id]['last_retrieve']
     end
 
+    def smart_action_allowed?(smart_actions_permissions) 
+      if !@smart_action_parameters||
+          !@smart_action_parameters[:user_id] ||
+          !@smart_action_parameters[:action_id] ||
+          !smart_actions_permissions ||
+          !smart_actions_permissions[@smart_action_parameters[:action_id]]
+        return false
+      end
+
+      @user_id = @smart_action_parameters[:user_id]
+      @action_id = @smart_action_parameters[:action_id]
+      @smart_action_permissions = smart_actions_permissions[@action_id]
+      @allowed = @smart_action_permissions['allowed']
+      @users = @smart_action_permissions['users']
+
+      return @allowed && (@users.nil?|| @users.include?(@user_id.to_i));
```

Comment:
[Fix] I intentionally cast the `@user_id` value to int to fit with the `@users` values type (which are of type int). 

Without this, we would retrieve a user id of type string from the JWT, to compare with user ids of type int ==> this never passes the include check (type mismatch).